R/pkg/R/DataFrame.R

```diff
@@ -2884,3 +2884,38 @@ setMethod("write.jdbc",
             write <- callJMethod(write, "mode", jmode)
             invisible(callJMethod(write, "jdbc", url, tableName, jprops))
           })
+
+#' randomSplit
+#'
+#' Return randomly split RDDs with the provided weights.
+#'
+#' @param x A SparkDataFrame
+#' @param weights a vector of weights for splits, will be normalized if they don't sum to 1
+#' @param seed Randomness seed value
+#'
+#' @family SparkDataFrame functions
+#' @rdname randomSplit
+#' @name randomSplit
+#' @export
+#' @examples
```

Comment:
Could you add `#' @note since 2.0.0` - we are trying to add that for 2.0, there should be some example already. I'm going to do a pass on that later - btw, you are welcome to take on that too!